docs/advanced_topics/accessibility_considerations.md

```diff
@@ -1,6 +1,6 @@
 # Accessibility considerations
 
-Accessibility for CMS-driven websites is a matter of [modeling content appropriately](content_modeling), [creating accessible templates](accessibility_in_templates), and [authoring accessible content](authoring_accessible_content) with readability and accessibility guidelines in mind.
+Accessibility for CMS-driven websites is a matter of [modeling content appropriately](content_modeling), [creating accessible templates](accessibility_in_templates) and [authoring accessible content](authoring_accessible_content) with readability and accessibility guidelines in mind.
```

Comment:
@Dev-Yusuf  no need for this change, we just had a PR add a bunch of Oxford commas and we just elected to leave it as is.

There is no formal guidance as to Whether we should or should not have these, so for now let's leave lines like this as is unless there is a critical readability problem.

```suggestion
Accessibility for CMS-driven websites is a matter of [modeling content appropriately](content_modeling), [creating accessible templates](accessibility_in_templates), and [authoring accessible content](authoring_accessible_content) with readability and accessibility guidelines in mind.
```